src/libraries/Ownable.sol

```diff
@@ -41,8 +49,10 @@ contract Ownable is IOwnable {
         @notice renounce ownership to 0 address
      */
     function renounceOwner() public virtual override onlyOwner {
+        newOwner = address(0);
```

Comment:
Would you mind removing this function as we discussed?

Reply:
right, check, 1sec

Reply:
done